src/pages/Home.tsx

```diff
@@ -253,33 +239,52 @@ function Home() {
                 Second Encrypted File
                 <Textarea value={doubleEncryptedFile?.toString()} />
               </FormLabel>
-              <Box>sarcoId: {sarcoId}</Box>
-              <Box>Recipient Address: {recipientAddress}</Box>
-              <Box>
-                Resurection TimeStamp: {resurrectionTime} {new Date(resurrectionTime).toString()}
-              </Box>
-              <Button onClick={() => initializeSarcophagus()}>Submit</Button>
-              <HStack>
-                <Box>Current (sarcoId, arweaveId):</Box>
-                <Box>{currentSarcoId}</Box>
-                <Box>{currentArweaveTxId}</Box>
-                <Box>Status: {sendStatus.status}</Box>
+              <Button
+                variant="solid"
+                bg="grey"
+                onClick={() => initializeSarcophagus()}
+              >
+                Initialize Sarcophagus
+              </Button>
+              <VStack
+                align="left"
+                border="2px"
+              >
+                <Box>Sarco Id: {currentSarcoId}</Box>
+                <Box>Arweave Tx Id: {currentArweaveTxId}</Box>
+                <Box>
+                  Status:{' '}
+                  {sendStatus.status === 'Pending'
+                    ? 'Waiting for Arweave TX to confirm'
+                    : sendStatus.status === 'Success'
+                    ? 'Arweave TX upload successful'
+                    : sendStatus.status}
```

Comment:
Anywhere that variables are checked against a specific string, we should setup enums to check these types rather than relying on strings. This helps enforce type correctness and prevents us from accidentally typing the wrong string if the variable is checked elsewhere.

```
enum ArweaveTxStatus {
  PENDING,
  SUCCESS,
  FAIL
}

...

sendStatus.status === ArweaveTxStatus.PENDING

```

Reply:
yeah, that's always a good idea. This code felt somewhat temporary, since, but I can add for sure.